docs/source/glossary.rst

```diff
@@ -19,6 +19,9 @@ Glossary
         server or another third-party data service, like a SQL database. Thus, catalogs form a hierarchy: any
         catalog can contain other, nested catalogs.
 
+    Catalog file
+        A specification file which contains a list of named entries describing how to load data sources.
```

Comment:
Would you mind specifying that this is in YAML format and link to the docs where that syntax is described?

Reply:
Sure i'll add that now